index.html

```diff
@@ -24,24 +24,23 @@
   <body>
     <header>
       <nav class="nav">
-        <a class="logo" href="#">Ermiyas Demsew</a>
-        <a
-          class="humburger"
-          href="javascript:void(0);"
-          class="icon"
-          onclick="myFunction()"
-        >
-          <i class="fa fa-bars"></i>
-        </a>
+        <a class="logo" href="#home">Ermiyas Demsew</a>
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
```

Comment:
- [ ] According to the requirements, You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). so I suggest adding some space (padding-right or margin-right) to the hamburger button, and center the hamburger button and the logo vertically to match your Figma design 👍

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/195068274-71bb954f-ae09-41c2-9f51-f7cc3cbb8ba6.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/195068166-5b052efc-a008-4c7d-a3d4-910f66ed8a9d.PNG" width="500" /> |

Reply:
I improved my work.

Reply:
Implemented changes 👍